tests/v2/validation/provisioning/rke2/etcd_backup_restore_test.go

```diff
@@ -66,6 +68,21 @@ func (r *RKE2EtcdSnapshotRestoreTestSuite) SetupSuite() {
 	client, err := rancher.NewClient("", testSession)
 	require.NoError(r.T(), err)
 
+	if clustersConfig.S3BackupConfig != nil {
```

Comment:
So I guess it doesn't matter that much at this point since there's gonna be a refactor by @slickwarren. But I don't agree with this mainly because of this: https://github.com/rancher/rancher/pull/41764#discussion_r1223332541. I don't think this etcd option is as wide as psact option currently. So for a single suite, making that change is unnecessary in my opinion.

Reply:
No this is what we suggested for Vivek for the time being. This will change with the refactor Caleb is working on